README.md

```diff
@@ -216,7 +216,7 @@ If unset, the two envs will have the following default values:
 
 -   `PARACHAIN_PALLET_ID`: `0x01`
 
--   `AUTHORIZE_UPGRADE_PREFIX`: `0x03`
+-   `AUTHORIZE_UPGRADE_PREFIX`: `0x02`
```

Comment:
The `README.md` file is generated from the `.adoc` file. Make sure to update the .adoc as well.

Reply:
If this https://github.com/chevdor/srtool-actions/pull/22/files#diff-e2d35707e12980263abed184f398892e5cafa7517654b2dc9f71b31f151aa37cR9  it's also updated.

Reply:
Indeed, my bad, all good, thank you